packages/scoutgame/src/referrals/updateReferralUsers.ts

```diff
@@ -90,5 +93,32 @@ export async function updateReferralUsers(referralCode: string, refereeId: strin
     return [referrer, referee];
   });
 
+  const referee = await prisma.scout.findUniqueOrThrow({
+    where: {
+      id: refereeId
+    },
+    select: {
+      displayName: true,
+      path: true
+    }
+  });
+
+  if (initialReferrer.email) {
+    await sendEmailTemplate({
```

Comment:
Can you catch this func? In case this fails I wouldn't want the saveOnboardingDetailsAction to fail.

Reply:
good point. updated